world.mysql.sql

```diff
@@ -0,0 +1,37 @@
+SET AUTOCOMMIT=0;
+
```

Comment:
This code is not specific to my contrib, that's why I think it shouldn't be in my sqltraced directory. Maybe I could add a `test` directory in the root directory and put the sql files in it?